dask/dataframe/tests/test_dataframe.py

```diff
@@ -2950,6 +2950,11 @@ def test_series_iteritems():
     for (a, b) in zip(df["x"].iteritems(), ddf["x"].iteritems()):
         assert a == b
 
+def test_series_iter():
+    s = pd.DataFrame({"x": [1, 2, 3, 4]})
+    ds = dd.from_pandas(s, npartitions=2)
+    for (a, b) in zip(iter(s["x"]), ds["x"].__iter__()):
```

Comment:
```suggestion
    for (a, b) in zip(s["x"], ds["x"]):
```

There is no reason to call `iter` or `__iter__` here (which are the same by the way).  The `zip` function should handle this automatically.